internal/configuration/validator/authentication.go

```diff
@@ -21,7 +21,7 @@ func ValidateAuthenticationBackend(configuration *schema.AuthenticationBackendCo
 	}
 
 	if configuration.File != nil {
-		validateFileAuthenticationBackend(configuration.File, validator)
+		ValidateFileAuthenticationBackend(configuration.File, validator)
```

Comment:
```suggestion
		validateFileAuthenticationBackend(configuration.File, validator)
```
Not sure if we really need to make that function public but if we are, why don't we put the LDAP one as well then?

Reply:
Nice spot, I changed it for testing something and will just change it back.